build-everything.proj

```diff
@@ -43,6 +43,10 @@
     <SetupProjects Include="setup/fsharp-setup-build.proj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BUILD_NUGET)'=='1'">
+    <NugetProjects Include="src/fsharp/fsharp.core.nuget/fsharp.core.nuget.proj" />
```

Comment:
Fix the capitalization here - "FSharp.Core" please! :)  Also search everything to check that "fsharp.core" is never used anywhere